libfreerdp/codec/h264_ffmpeg.c

```diff
@@ -241,14 +256,14 @@ static int libavcodec_compress(H264_CONTEXT* h264, BYTE** ppDstData, UINT32* pDs
 #if LIBAVUTIL_VERSION_INT >= AV_VERSION_INT(52, 48, 100)
 	sys->videoFrame->colorspace = AVCOL_SPC_BT709;
 #endif
+	sys->videoFrame->chroma_location = AVCHROMA_LOC_LEFT;
```

Comment:
Same here, please add a version check to disable on older versions
LIBAVUTIL 52.92.100